### PR TITLE
Exploring the future Software Development Lifecycle (SDLC) flow

### PR DESCRIPTION
**Pull request by wpschaub**
Looking for Insights and Feedback from other pioneers of continuous SDLC improvements.

### CODE REVIEW

File: content/engineering-practices-sdlc-improvements.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+- Multiple environments that can diverge from each other in terms of data, setup, 3rd party software, for example .NET, and our solution artifacts.
+- Multiple environments that increase **complexity** and **total cost of ownership**.
+
+---
+
+# SDLC / FLOW / FUTURE
+
+> ![SDLC of the future](../images/engineering-practices-sdlc-improvements-2.png) 
+
+We want to move as much of our quality and security scans to the left as we can, adopt a mindset of continuous delivery and release on demand, and decrease the number of environments. We want to gradually show features in a new (green) version, while keeping the live production (blue) version. That way we can switch from green to blue and switch back if needed.
```
Comment (MerlinDarkstar) [inline content/engineering-practices-sdlc-improvements.md]: ... scan to the left as we can, ... 
should be:
...scan to the left as far as we can, ...

**Review by MerlinDarkstar — DISMISSED** [review]

File: content/engineering-practices-sdlc-improvements.md (hunk 1 above)
Comment (wsbctechnicalblog) [reply content/engineering-practices-sdlc-improvements.md]: Fixed, thanks for catching that.

**Review by MerlinDarkstar — DISMISSED** [review]

**Review by danielrbroderick — DISMISSED** [review]

**Review by AndreKaminski — CHANGES_REQUESTED** [review]
Can I suggest that you remove the references to system test? The problem is that the diagram might be revealing our configuration, which sec will not be happy about. This structure that we have here is pretty unique to WSBC, as other organizations don't have systest environment but rather the basic 3 dev, stage, production. I think this should be good enough for you to make the point

**Review by AndreKaminski — CHANGES_REQUESTED** [review]
I still see these references here "user acceptance testing are conducted in the **system test environment**, additional user acceptance testing and ad-hoc bug analysis are carried out in **Staging**, and we are all familiar with what happens in Production"

**Review by littleswancode — DISMISSED** [review]

**Review by danielrbroderick — DISMISSED** [review]

**Review by danielrbroderick — APPROVED** [review]

**Pull request merged**